Atomic/install.py

```diff
@@ -132,6 +160,42 @@ def install(self):
             # using the backend approach vs the atomic args
             be.syscontainers.set_args(self.args)
             return be.install(self.image, self.name)
+        if util.InstallData.image_installed(img_obj):
+            raise ValueError("Image {} is already installed.".format(self.image))
+
+        installation = None
+        if storage == 'docker' and not args_system:
+            if self.args.system_package == 'build':
+                raise ValueError("'--system-package=build' is not supported for docker backend")
+            labels = self.get_labels()
```

Comment:
We should be using the refactored approach to this where you get labels from the image_object.

iirc img_object.get_label or img_object.labels